app/models/ability.rb

```diff
@@ -146,57 +148,73 @@ def signed_in_with_roles(user)
     end
   end
 
-  def signed_in_with_organizer_role(user)
-    # ids of all the conferences for which the user has the 'organizer' role
-    conf_ids_for_organizer = Conference.with_role(:organizer, user).pluck(:id)
-
-    can :manage, Resource, conference_id: conf_ids_for_organizer
-    can [:new, :create], Conference if user.has_role?(:organizer, :any)
-    can :manage, Conference, id: conf_ids_for_organizer
-    can :manage, Splashpage, conference_id: conf_ids_for_organizer
-    can :manage, Contact, conference_id: conf_ids_for_organizer
-    can :manage, EmailSettings, conference_id: conf_ids_for_organizer
-    can :manage, Campaign, conference_id: conf_ids_for_organizer
-    can :manage, Target, conference_id: conf_ids_for_organizer
+  def signed_in_with_organization_admin_role(user)
+    org_ids_for_organization_admin = Organization.with_role(:organization_admin, user).pluck(:id)
+
+    can :manage, Organization, id: org_ids_for_organization_admin
+    can :new, Conference
```

Comment:
Since, a user can be an org admin of many organizations, I thought we can allow choosing the organization in the `conference#new` page. That way, the permission for the `:new` action shall be given to all organization admins. However, in the form, they will get a drop down of only those organizations in which they belong to.